contribs/gmf/src/services/objecteditingquery.js

```diff
@@ -220,3 +220,17 @@ gmf.ObjectEditingQuery.GetQueryableLayerNodesResponse;
  * }}
  */
 gmf.ObjectEditingQuery.QueryableLayerInfo;
+
```

Comment:
`@struct` and `@constructor` are not for typedefs.
Does it work to put an `@export` on a typedef?